src/vendor/reconnecting-websocket/reconnecting-websocket.js

```diff
@@ -65,17 +65,13 @@
  * =======
  * Options can either be passed upon instantiation or set after instantiation:
  *
- * var socket = new ReconnectingWebSocket(url, null, { debug: true, reconnectInterval: 4000 });
+ * var socket = new ReconnectingWebSocket(url, null, { reconnectInterval: 4000 });
```

Comment:
where did you find this changes? Don't see in https://github.com/joewalnes/reconnecting-websocket/

Reply:
I made them, and yes I remember what you said like 3 months ago about vendored code updates but I think this is a change for the better so its actually synced now

Reply:
Is there any improvements? You can pass `process.env.NODE_ENV !== 'production'` as `debug` to replicate the logic you are trying to introduce.

I'd prefer to keep vendor untouched for the same reasons as before. Even though there were no updates to the library in years.

Reply:
Right, but before the it wasn't passed, and now it doesn't need to be.  It's dead code, and if it does need to be updated, we can update around it (an extra minute at most).  I'll revert it for now to get this in though.